Makefile

```diff
@@ -11,7 +11,7 @@ endif
 
 GRPC_SERVICES_DIR=src/grpc/autogen
 
-GATEWAY_RS_VSN ?= "v1.0.0-alpha.23"
+GATEWAY_RS_VSN ?= "e36e9ffd1fe375a66cdcf46ceafc73519e4e53e0"
```

Comment:
Does gateway-rs need a new tag?

Reply:
https://github.com/helium/miner/pull/1527 Would take care of this when merged into the poc branch